client/src/pages/private/ParticipantTable.js

```diff
@@ -233,7 +233,7 @@ const ParticipantTable = () => {
     setActiveModalForm(participantEngageStatus.MULTI_SELECT_SITE);
   };
 
-  const handleEngage = async (participantId, status, additional = {}, participantInfo = null) => {
+  const handleEngage = async (participantId, status, additional = {}, _participantInfo = null) => {
```

Comment:
Can we just remove this parameter?

Reply:
I wonder the same, not sure at what stage it became optional. But it's being passed on for every `handleEngage`